Favor/Favor/Sources/Scenes/Application/SceneDelegate.swift

```diff
@@ -24,3 +26,33 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
 		self.appCoordinator?.start()
 	}
 }
+
+private extension SceneDelegate {
+  func setupNavigationBarAppearance() {
+    let appearance = UINavigationBarAppearance()
+    let backButtonAppearance = UIBarButtonItemAppearance()
+    
+    let leftArrowImage = UIImage(named: "ic_leftArrow")?
+      .withRenderingMode(.alwaysOriginal)
+      .withAlignmentRectInsets(UIEdgeInsets(top: 0, left: 0, bottom: 10, right: 0))
+    
+    backButtonAppearance.normal.titleTextAttributes = [
+      .foregroundColor: UIColor.clear
+    ]
+    
+    appearance.setBackIndicatorImage(leftArrowImage, transitionMaskImage: leftArrowImage)
+    appearance.backButtonAppearance = backButtonAppearance
+    
+    appearance.titleTextAttributes = [
+      .foregroundColor: UIColor.favorColor(.typo),
+      .font: UIFont.favorFont(.bold, size: 18)
+    ]
+    
+    appearance.backgroundColor = .favorColor(.background)
```

Comment:
```suggestion
    appearance.backgroundColor = .clear
```

![스크린샷 2023-01-22 10 48 00](https://user-images.githubusercontent.com/60438045/213897058-d341aa14-fc34-4aee-b08e-c3939332c32e.png)

디자인에 친구 페이지 보면 `backgroundColor`는 `.clear`가 좋을 것 같아
배경 색상 따라가는걸 원하시는 것 같아서!